integration/integration_test.go

```diff
@@ -53,7 +53,15 @@ func init() {
 	flag.BoolVar(&doDelete, "eksctl.delete", true, "Skip the cleanup after the tests have run")
 	flag.StringVar(&kubeconfigPath, "eksctl.kubeconfig", "", "Path to kubeconfig (default: create it a temporary file)")
 	flag.StringVar(&privateSSHKeyPath, "eksctl.git.sshkeypath", defaultPrivateSSHKeyPath, fmt.Sprintf("Path to the SSH key to use for Git operations (default: %s)", defaultPrivateSSHKeyPath))
+}
+
+func TestSuite(t *testing.T) {
+	parseTestArgs()
+	testutils.RegisterAndRun(t)
+}
 
+func parseTestArgs() {
+	// go1.13+ testing flags regression fix: https://github.com/golang/go/issues/31859
```

Comment:
Turns out they had a fix out which was reverted later because it didn't fully address the issue: https://github.com/golang/go/commit/5e21032b3d179a8bc0d9e70e4725629defad9d8b